packages/broker/test/integration/plugins/storage/Storage.test.ts

```diff
@@ -37,7 +37,7 @@ export function buildMsg({
 }): StreamMessage {
     return new StreamMessage({
         messageId: new MessageID(toStreamID(streamId), streamPartition, timestamp, sequenceNumber, publisherId, msgChainId),
-        content: JSON.stringify(content),
+        content: utf8ToBinary(JSON.stringify(content)),
```

Comment:
We can just remove the content parameter and use 

```suggestion
        content: new Uint8Array()
```